dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -1008,6 +1010,105 @@ public static void label(List<String> args) {
         }
     }
 
+    public static void versionTag(List<String> args) {
+        if (args.size() > 0) {
+            final String toolpath = reqVal(args, "--entry");
+            try {
+                Container container = containersApi.getContainerByToolPath(toolpath);
+                long containerId = container.getId();
+                if (args.contains("--add")) {
+                    if (container.getMode() != ModeEnum.MANUAL_IMAGE_PATH){
+                        kill("Only manually added images can add version tags");
+                    }
+
+                    final String tagName = reqVal(args, "--add");
+                    final String gitReference = reqVal(args, "--git-reference");
+                    final Boolean hidden = Boolean.valueOf(optVal(args, "--hidden", "f"));
+                    final String cwlPath = optVal(args, "--cwl-path", "/Dockstore.cwl");
+                    final String dockerfilePath = optVal(args, "--dockerfile-path", "/Dockerfile");
+                    final String imageId = reqVal(args, "--image-id");
+
+                    final Tag tag = new Tag();
+                    tag.setName(tagName);
+                    tag.setHidden(hidden);
+                    tag.setCwlPath(cwlPath);
+                    tag.setDockerfilePath(dockerfilePath);
+                    tag.setImageId(imageId);
+                    tag.setReference(gitReference);
+
+                    List<Tag> tags = new ArrayList<>();
+                    tags.add(tag);
+
+                    List<Tag> updatedTags =  containerTagsApi.addTags(containerId, tags);
+
+                } else if (args.contains("--update")) {
+                    final String tagName = reqVal(args, "--update");
+                    List<Tag> tags = container.getTags();
+
+                    for (int i = 0; i < tags.size(); i++) {
+                        Tag tag = tags.get(i);
```

Comment:
Since you're not doing anything with `i` here, change this to an enhanced for loop or a forEach